README.md

```diff
@@ -1,7 +1,5 @@
-# curriculum-databases-projects-template
+# Vet-Clinic
 
```

Comment:
- [OPTIONAL] I think it can be a good idea if you add a short description of your project so that your README can looks  more professional 😎